src/main/java/fact/coordinates/CoordinateUtils.java

```diff
@@ -0,0 +1,32 @@
+package fact.coordinates;
+
+
+import java.time.Duration;
+import java.time.ZoneId;
+import java.time.ZonedDateTime;
+
+/**
+ * Created by maxnoe on 24.02.17.
+ */
+public class CoordinateUtils {
+    // reference datetime
+    private static final ZonedDateTime gstReferenceDateTime = ZonedDateTime.of(2000, 1, 1, 12, 0, 0, 0, ZoneId.of("UTC"));
+
+
+    /**
+     * Convert a DateTime object to greenwhich sidereal time according to
+     * https://en.wikipedia.org/wiki/Sidereal_time#Definition
+     * @param datetime
+     * @return gst in radians
+     */
+    public static double datetimeToGST(ZonedDateTime datetime){
```

Comment:
If I see this correctly, this only ever gets used in objects of type CelestialCoordinate. In that cas eit makes sense to move that function to the interface as has been done with the greatCircleDistance method.